test/acceptance/auth.js

```diff
@@ -15,13 +15,13 @@ describe('vkAuth', function() {
             expect(token.access_token).to.be.a('string');
             done();
         })
-    })
+    });
 
     it('should return error about invalid credentials', function(done) {
         this.auth.authorize('user@example.com', 'password', function(err, token) {
             expect(token).to.be.undefined;
-            expect(err.message).to.equal('Incorrect login or password\n');
+            expect(err.message).to.equal('Incorrect login or password\r\n');
```

Comment:
With this line tests pass perfectly on Windows, but fail on Linux/Mac because of different new line characters. The best solution here will be to remove this symbols at all. Will merge pull request and work on fixing this issue, thank you